### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by massimilianobraglia**
I have written a first version of `README.md` file plus updated LICENSE copyright year. Give it a look!
`README.md` for single component will come as soon as I have time to write those down!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Jymfony is a **Node.js framework** for applications and a set of reusable **Javascript components**.
+Jymfony was born from an idea of [Alessandro Chitolina](@alekitto) as a port of the popular PHP framework, Symfony.
+
+Requirements
+------------
+- Node.js >= 6.0.0
+
+Installation
+------------
+The suggested installation method is via [npm](https://npmjs.com/):
```
Comment (alekitto) [inline README.md]: `yarn` is the preferred package manager

**Comment by alekitto** [comment]
Thanks ❤️

**Comment by massimilianobraglia** [comment]
It comes in a sec! 👍 :)

**Comment by massimilianobraglia** [comment]
Done!

**Pull request merged**